snuba/datasets/storages/__init__.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class StorageKey(Enum):
```

Comment:
Please add a docstring to explain what a StorageKey is and which constraints it has (like that it has to be unique per storage class)